modules/oc-mirror-disk-to-mirror.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-disk-to-mirror_{context}"]
+= Mirroring from disk to mirror
+
+You can use the oc-mirror plug-in to mirror the contents of a generated image set to the target mirror registry.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`) in the disconnected environment.
+* You have installed the `oc-mirror` CLI plug-in in the disconnected environment.
+* You have generated the image set file by using the `oc mirror` command.
+* You have transferred the image set file to the disconnected environment.
+// TODO: Confirm prereq about not needing a cluster, but need pull secret misc
+
+.Procedure
+
+* Run the `oc mirror` command to process the image set file on disk and mirror the contents to a target mirror registry:
++
+[source,terminal]
+----
+$ oc mirror --from=./mirror_seq1_000000.tar \ <1>
+  docker://registry.example:5000/             <2>
```

Comment:
the docker url should be: "docker://registry.example:5000"  or "docker://registry.example:5000/namespace",  if no namespace should remove '/'

Reply:
Thanks, updated